lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -235,6 +235,13 @@ def bash_systemd_polling_functions
         return command
       end
 
+      def get_ssh_group(ssh_user)
```

Comment:
If we have access to the machine, why not actually determine the group for the user?